services/weblate/weblate-user-statistics.tester.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const { ServiceTester } = require('../tester')
+const { isMetricWithPattern } = require('../test-validators')
+
+const t = (module.exports = new ServiceTester({
+  id: 'weblate',
+  title: 'Weblate',
+}))
```

Comment:
Same feedback as the tester for the entity count. Can't use the helper due to the multiclass exports but should be able to give this some better values